docs/Makefile

```diff
@@ -35,14 +35,14 @@ lint: lint-fix
 #
 # - .markdownlintignore holds the configuration for files to be ignored
 # - .markdownlint.yaml contains the rules for markdownfiles
-#
 # renovate: datasource=docker depName=ghcr.io/igorshubovych/markdownlint-cli
 MDL_DOCKER_VERSION := v0.33.0
-MDL_CMD := docker run -v $(ROOT_DIR)../:/workdir --rm  ghcr.io/igorshubovych/markdownlint-cli:$(MDL_DOCKER_VERSION)  "**/*.md" 
+#MDL_CMD := docker run -v $(ROOT_DIR)../:/workdir --rm  ghcr.io/igorshubovych/markdownlint-cli:$(MDL_DOCKER_VERSION)  "**/*.md" 
+MDL_CMD := docker run -v $(ROOT_DIR)../:/workdir --rm 
 
 .PHONY: markdownlint markdownlint-fix
 markdownlint:
-	$(MDL_CMD)
+	$(MDL_CMD) davidanson/markdownlint-cli2-rules:next "**/*.md" 
 
 markdownlint-fix:
-	$(MDL_CMD) --fix
+	$(MDL_CMD) --entrypoint="markdownlint-cli2-fix" davidanson/markdownlint-cli2-rules:next "**/*.md" 
```

Comment:
can we use pinned version tags for the image here?

Reply:
since we opened this pr there was finally the official release, and we do not need to use next anymore, will adapt it

Reply:
i will drop my commit with the applied changes - and add a new one - this should make things easier

Reply:
markdownlint-cli2 is still not release https://hub.docker.com/r/davidanson/markdownlint-cli2-rules/tags - i added a comment also for the renovate update :) we need to delete a line with the next version